<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nome_input">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Insira Nome</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data Nascimento</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="data_input"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Telefone</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="telefone_input">
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Insira Telefone</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>CPF</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="cpf_input">
     <property name="inputMask">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Insira CPF</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="email_input">
     <property name="placeholderText">
      <string>Insira Email</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="senha_input">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Digite Senha</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Confirmar Senha</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="confirmar_senha_input">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Digite Novamente </string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Ocupação</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="recepcionista_check_input">
       <property name="text">
        <string>Recepcionista</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="veterianrio_check_input">
       <property name="text">
        <string>Veterinário(a)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QPushButton" name="registrarBTN">
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
